src/Microsoft.Docs.Template/Models/Conceptual.cs

```diff
@@ -7,6 +7,7 @@ namespace Microsoft.Docs.Build
     //   Conceptual model is more than just an html string, it also contain other properties.
     //   We currently bake these other properties into PageModel to make the output flat,
     //   but it makes conceptual kinda special. We may consider lift them outside PageModel.
+    [PageSchema]
```

Comment:
>PageSchema [](start = 5, length = 10)

what if the contributor is false here but the configuration says we need contributor?

Reply:
It'll not have contributors.

---
In reply to: [213583619](https://github.com/dotnet/docfx/pull/3316#discussion_r213583619) [](ancestors = 213583619)